packages/dx-react-grid/docs/reference/table-filter-row.md

```diff
@@ -51,11 +54,51 @@ column | [Column](grid.md#column) | A column.
 filteringEnabled | boolean | Specifies whether filtering by a column is enabled.
 getMessage | ([messageKey](#localization-messages): string) => string | Returns the filter editor placeholder text. Available in the "@devexpress/dx-react-grid-material-ui" package.
 
+### TableFilterRow.FilterSelectorProps
+
+Describes properties passed to a component that renders a filter selector.
+
+Field | Type | Description
+------|------|------------
+iconComponent | ComponentType&lt;[TableFilterRow.IconProps](#tablefilterrowiconprops)&gt; |  A component that renders icons.
+value | string | The currently selected filter operation.
+availableValues | Array&lt;string&gt; | The list of available filter operations.
+onChange | (value: string) => void | Handles filter operation changes.
+
+### TableFilterRow.IconProps
+
+Describes properties passed to a component that renders icons for the filter selector.
+
+Field | Type | Description
+------|------|------------
+type | string | Specifies the type of icon to be used.
+
+### TableFilterRow.EditorProps
+
+Describes properties passed to a component that renders a filter editor.
+
+Field | Type | Description
+------|------|------------
+value | any | The current editor value.
+disabled | boolean | Specifies whether the editor is disabled.
+onChange | (value: string) => void | Handles filter value changes.
+getMessage | ([messageKey](#localization-messages): string) => string | Returns a specified localization message.
+
 ## Localization Messages
 
 Field | Type | Default | Description
 ------|------|---------|------------
 filterPlaceholder? | string | 'Filter...' | The filter editor placeholder text. Available in the "@devexpress/dx-react-grid-material-ui" package.
+contains? | string | 'Contains' | The 'contains' filter operation name.
```

Comment:
`filterPlaceholder` has old description -  ... Available in the "@devexpress/dx-react-grid-material-ui" package.